moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -276,12 +273,12 @@ void ChompOptimizer::registerParents(const moveit::core::JointModel* model)
     {
       if (model->getParentLinkModel() == nullptr)
       {
-        ROS_ERROR_STREAM("Model " << model->getName() << " not root but has NULL link model parent");
+        RCLCPP_ERROR(LOGGER2, "Model %s not root but has NULL link model parent", model->getName());
```

Comment:
```suggestion
        RCLCPP_ERROR(LOGGER2, "Model %s not root but has NULL link model parent", model->getName().c_str());
```
Same in the ERROR below to make CI happy